webapps-schema/src/main/resources/schema/elasticsearch/create/index/camunda-tenant.json

```diff
@@ -17,8 +17,11 @@
       "description": {
         "type": "text"
       },
-      "memberKey": {
-        "type": "long"
+      "memberId": {
+        "type": "keyword"
+      },
+      "memberType": {
+        "type": "keyword"
```

Comment:
We probably need the same change for the opensearch mapping.

Reply:
Good catch, added :)